lib/src/ui/movie_detail.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'movie_image.dart';
+import '../utils/numeric_constants.dart';
+import '../utils/string_constants.dart';
+import '../utils/styles.dart';
+
+class MovieDetail extends StatelessWidget {
```

Comment:
`MovieDetailPage` or `MovieDetailScreen` would be a good or better name right?

Reply:
Yes, thank you. Fixed